packages/web/src/components/list/ToggleButton.js

```diff
@@ -253,10 +254,10 @@ class ToggleButton extends Component {
 		return (
 			<Container
 				style={this.props.style}
-				className={`${toggleButtons} ${this.props.className || ''}`}
+				css={[toggleButtons, this.props.className || '']}
```

Comment:
Just confirming here, this is what toggleButtons looks like: 

```const toggleButtons = css`some_styles`;```

I was checking this and here the issue is that `css` prop only accepts the styles created by emotion and if we pass an external className consider a string for now. It will create an invalid CSS property. 

For more info check this: https://codesandbox.io/s/interesting-elgamal-lorko?file=/src/App.js